conda_forge_tick/make_migrators.py

```diff
@@ -97,6 +98,29 @@
 MAX_PR_LIMIT = 50
 MAX_SOLVER_ATTEMPTS = 50
 CHECK_SOLVABLE_TIMEOUT = 90  # 90 days
+DEFAULT_MINI_MIGRATORS = [
```

Comment:
Thanks for this!

What was the reason to not add `StdlibMigrator` to the default set? It's in use for both types of migrators (after #2372 & #2459)